view_all_inc.php

```diff
@@ -111,10 +111,15 @@
 		<div class="btn-toolbar">
 			<div class="btn-group pull-left">
 		<?php
-			# -- Print and Export links --
+			# -- Print link --
```

Comment:
I prefer a second threshold as the workflow of the HTML/Word exports is different from the CSV/Excel exports.
@vboctor what do you think?